docs/odoo/usage/debug-assets-mode.rst

```diff
@@ -0,0 +1,19 @@
+================================
+How to activate debug assets mode 
+================================
```

Comment:
Fix format. https://odoo-development.readthedocs.io/en/latest/dev/docs/rst-requirements.html#titles-and-sections